packages/interface-ipfs-core/README.md

```diff
@@ -53,16 +62,16 @@ Include this badge in your readme if you make a new module that implements inter
 
 In JavaScript land:
 
-```js
-npm install interface-ipfs-core
+```console
```

Comment:
```suggestion
```bash
```

Reply:
Bash does bash script syntax highlighting which can result in weird output when you're trying to format console commands.

Reply:
bash:

```bash
$ npm install interface-ipfs-core
...some output
$ git clone https://github.com/ipfs/js-ipfs-http-client
$ npm install
$ npm test
```

console (similar but the command output is highlighted):

```console
$ npm install interface-ipfs-core
...some output
$ git clone https://github.com/ipfs/js-ipfs-http-client
$ npm install
$ npm test
```

Reply:
ah cool didn't know